contracts/gasToken/GasTokenForwarder.sol

```diff
@@ -88,35 +134,35 @@ contract GasTokenForwarder is Ownable {
 
     }
 
-    function mintGasToken( uint256 mint ) public {
+    function mintGasToken( uint256 mint ) public onlyOwner {
         _gasToken.mint(mint);
     }
 
     function balanceOfGasToken(address who) external view returns (uint256){
         return _gasToken.balanceOf(who);
     }
 
-    function freeGasToken(uint256 value) public returns (bool success){
```

Comment:
Make private methods public with onlyOwner modifier that is to be used from outside